doc/using/configuration.chapter.md

```diff
@@ -189,6 +189,13 @@ The following attributes can be passed in [`config`](#chap-packageconfig).
 <include xmlns="http://www.w3.org/2001/XInclude" href="../doc-support/result/config-options.docbook.xml"/>
 ```
 
+## Calling Nixpkgs from the Module System {#sec-invoke-nixpkgs-by-module}
+
+For [module system](#module-system) application authors, Nixpkgs provides a reusable module for the purpose of invoking Nixpkgs. Most users should not have to be aware of this. The documentation for its options is meant to be re-exposed in the application's documentation and is therefore not repeated here.
+
+The flake attribute for this module is `.modules.generic.nixpkgs`.
+
+Non-flake applications may import the implementation file [`"${nixpkgs}/pkgs/top-level/module/module.nix"`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/module/module.nix).
```

Comment:
```suggestion
Applications can import the implementation file [`"${nixpkgs}/pkgs/top-level/module/module.nix"`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/module/module.nix).

The flake attribute (experimental) for this module is `.modules.generic.nixpkgs`.
```

I would oppose introducing the same "non-X" terminology as has proliferated for NixOS (calling everything outside NixOS a "non-NixOS" system as if NixOS was the only thing that existed). Also flakes should not be documented as if they were stable and the first thing to consider even thinking about.